testing/unit/renderer/specs/pages/LandingPage.spec.js

```diff
@@ -0,0 +1,72 @@
+import { createVue } from '../../utils/VueUtils';
+import { EVENTS, DIALOGS } from '@/contants';
+import LandingPage from '@/renderer/components/pages/LandingPage.vue';
+
+describe('LandingPage.js', () => {
+  const sendEvent = LandingPage.methods.sendEvent;
+
+  afterEach(() => {
+    LandingPage.methods.sendEvent = sendEvent;
+  });
+
+  it('should mount and render', () => {
+    const vm = createVue({
+      el: document.createElement('div'),
+      render: h => h(LandingPage)
+    }).$mount();
+    expect(vm.$el.querySelector('.name').textContent).to.equal('SpringRoll Studio');
+  });
+
+  it('should dispatch EVENTS.OPEN_DIALOG with parameter DIALOGS.PROJECT_LOCATION_SETTER', (done) => {
+    LandingPage.methods.sendEvent = (event, type) => {
```

Comment:
Currently, if this test is broken it doesn't fail, but rather times out (which is a failure I guess...). Can we instead use a  ninon stub or something here and assert based on if it's called? Something like:

```javascript
LandingPage.methods.sendEvent = sinon.stub();

// .. do stuff

expect(
  LandPage.methods.sendEvent.calledWith(EVENTS.OPEN_DIAGLOG, DIALOGS.PROJECT_LOCATION_SETTER)
).to.equal(true);